Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 ruby '2.1.3'
 
 gem 'sinatra'
-gem 'cocoapods-core'
+gem 'cocoapods-core', '>= 0.a'
```

Comment:
Oh, it allows pre-release. Wasn't clear because lack of semver in RubyLand.

Perhaps it would be a good idea to pin this to ~> 1.0 once it's final:

```ruby
gem 'cocoapods-core', '~> 1.0'
```